<launch>
  <node pkg="necst" name="antenna_move" type="S_ROS_antenna_move.py" output="screen"/>
  <node pkg="necst" name="encoder_status" type="S_ROS_encoder.py"/>
  <node pkg='necst' name='dome' type='S_ROS_dome.py'/>
  <node pkg='necst' name='abs_controller' type='S_ROS_abs.py'/>
  <node pkg='necst' name='m4_controller' type='S_ROS_m4.py'/>
  <node pkg='necst' name='m2_controller' type='S_ROS_m2.py'/>
  <node pkg='necst' name='drive' type='S_ROS_drive.py'/>
</launch>
